oslabs-beta/KataLog
# search bug resolved

##Overview
**Issue Type**
- [] Feature
- []Tech debt

**Description**
Bug that was not rendering the `searchedLogs` is now resolved.

**Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix**
Navigate to the `Logs` page
Click on `search` icon and search for logs
Confirm that `logs` are rendered without an error

**Previous Behavior**
The `searchLogs` were identifying the `_id` from MongoDB and did not know how to treat this in the line `if (key !== 'podInfo')...`

**Expected Behavior**
Logs now render properly

**How Has This Been Tested**
See **Steps to Reproduce Bug/Validate Feature/Confirm Tech Debt Fix**

**Screenshots & Videos**
N/A

**Additional Context**
N/A
